graviti/utility/user.py

```diff
@@ -355,3 +357,145 @@ def update(self, __m: Any = (), **kwargs: _V) -> None:  # pylint: disable=argume
 
         """
         self._data.update(__m, **kwargs)
+
+
+class OrderedDict(MutableMapping[str, _V], UserMapping[str, _V]):
```

Comment:
`UserOrderedDict`？

Reply:
If you need to rewrite the `__getitem__` method, you should not set `UserMapping` as base class